packages/rich-text/src/component/index.native.js

```diff
@@ -408,9 +427,8 @@ export class RichText extends Component {
 
 		// We know for certain that on focus, the old selection is invalid. It
 		// will be recalculated on `selectionchange`.
-		const index = undefined;
 
-		onSelectionChange( index, index );
+		onSelectionChange( this.selectionStart, this.selectionEnd );
```

Comment:
Curious if this is related to mentions or if there is a specific case that this update is addressing? @SergioEstevao

Reply:
When doing insert of the mentions we found a crash. It was related to the edge case where those indexes were set to NaN and then send to iOS native view. 

That didn't like it and make it crash, I think this new code brought this edge case up and so I did this fix.

Reply:
Makes sense, thanks for the explanation!